aequilibrae/paths/parameters.pxi

```diff
@@ -15,6 +15,6 @@ cdef ITYPE_t NULL_IDX = 18446744073709551615
 cdef double INFINITE = 1.79769313e+308
 
 VERSION = 0.6
-MINOR_VRSN = 2
+MINOR_VRSN = 3
 binary_version = "0.6.1"
```

Comment:
is the binary version different from ```str(VERSION) + "." + str(MINOR_VRSN)```

Reply:
Yes. This is because of AequilibraE for QGIS, which gets the Python package delivered with it and downloads the binaries manually.